### PR TITLE
Add storybook

### LINKED ISSUE #138: Add Storybook
Add [Storybook](https://storybook.js.org/). Now that we're getting into visual UI dev instead of set-up, this will help speed up component design & dev. It lets you create components in an isolated environment with hot-reload. Kinda like living documentation for your components but also a big help for developing

### PR DESCRIPTION
**Pull request by jamesvclements**
Resolves #138 

You can check out the README for instructions to run the storybook, it's just `npm install; npm run storybook;` and hitting [localhost:9001](http://localhost:9001) 🎆

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,23 @@ Web framework for hobbyist and freelance bartenders 🍸
...
   * [Development](#ui-development)
+    * [Developing with Storybook](#ui-development-with-storybook)
+    * [Integrated Development](#integrated-ui-development)
 * [API](#api)
   * [Development](#api-development)
 
 ## UI
 ### UI Development
-*These instructions assume you're in the `/bartop-ui` directory*
+*These instructions assume you're in the `/bartop-ui` directory*.
```
Comment (jamesvclements) [inline README.md:16]: ^ I truly have no idea why I did that

Comment (dpopp07) [reply README.md:16]: lmao

**Review by dpopp07 — APPROVED** [review]
Triggered because this reminds me of Carbon which reminds me of SLAM but all looks good to me.

**Comment by jamesvclements** [comment]
C u @ Lone Star Court

**Pull request merged**